packages/@react-native-windows/cli/README.md

```diff
@@ -37,7 +37,7 @@ Options:
 | `--buildLogDirectory` | Optional directory where MSBuild log files should be stored | [string] |
 | `--info`              | Dump environment information | [boolean] |
 | `--direct-debugging`  | Enable direct debugging on specified port | [number] |
-| `--useWinUI3`         | Targets WinUI 3.0 (Preview) instead of UWP XAML. | [boolean] |
+| `--useWinUI3`         | Targets WinUI 3 (through the [Windows App SDK](https://docs.microsoft.com/windows/apps/windows-app-sdk/)) instead of UWP XAML. | [boolean] |
```

Comment:
This and the below are parameters of `react-native-windows-init`, instead of `run-windows`. Not sure what they are doing here, but the flag does not work when using the command the package documents.

Reply:
Looks like it was added there in May as a fix for issue #7426. Is there a different README that should've been updated instead?

Reply:
...or just reformat to make it clear which one's are for `react-native-windows-init` and which are for `run-windows`?

Reply:
yeah I think this was the wrong change to make unless I'm missing something. @dannyvv looks like you reviewed that PR #7788, do you agree we should not have those options in the package CLI readme?

@AgneLukoseviciute here is the readme for the init CLI: https://github.com/imrishabh18/react-native-windows/blob/13b1703ca14cb1528ae522e04e944f0122585f0b/packages/react-native-windows-init/README.md

Reply:
fixed.